gymnasium/dataclasses.py

```diff
@@ -0,0 +1,16 @@
+from dataclasses import dataclass
```

Comment:
This file was needed (I think) because putting it in `registration.py` or `core.py` both cause a circular import error for some reason - will need to think on this.

Reply:
Still unsure on any better alternative

Reply:
sticking with this (renamed file though)